app/src/com/snapyr/flappybird/SnapyrComponent.kt

```diff
@@ -26,20 +26,30 @@ class SnapyrComponent (private val context: Context) {
     var singleton:SnapyrData= SnapyrData.instance;
 
     internal fun onDoIdentify() {
-
-
-//        Log.d("onDoIdentify",singleton.identifyKey)
-
         Snapyr.with(context).identify(singleton.identifyUserId)
         Snapyr.with(context).identify(Traits().putName(singleton.identifyName))
         Snapyr.with(context).identify(Traits().putEmail(singleton.identifyEmail))
     }
 
+
     internal fun onDoTrack() {
         Log.d("onDoTrack", "Track tapped")
         Snapyr.with(context).track("ClickPlay")
     }
 
+
+    internal fun yourScore(score: Int) {
+        Log.d("onDoTrack", "Track tapped")
+        if(score % 2 == 0)
+            Snapyr.with(context).track("reachedascoreof_$score")
+    }
+
+
+    internal fun onScore2SendEmail() {
```

Comment:
rename the event here to just "score" and add a property for "total", whose value is 2.  then, remove the channel from the method name because that logic is in the campaign. it should not be in the app.